### DISCUSSION TITLE
Consolidating data from multiple instances for the same metric

### DISCUSSION THREAD

**Discussion by s-kuchi** [NetApp/harvest#2761] [Q&A] [answered]
@cgrinds @rahulguptajss : Because of upgrading Harvest and cleaning up dead-instances over an year, I ended up multiple metrics for the same aggregate or volume from multiple instances and multiple nodes. As you know, this would create different streams of data although for different timeframes. In the case below, a 180 day period, shows 3 data streams at different points in time.

Can you give me any suggestions as to consolidate this data based on one metric say "aggr" in this case? I tried outerjoin without success.

![image](https://github.com/NetApp/harvest/assets/133256656/1ddf71a0-45e2-40ef-a3a7-f6608b5dd720)

**Comment by rahulguptajss** [comment, marked as answer]
@s-kuchi See if below helps

```
max(max_over_time(volume_read_latency[180d])) without (node, instance) 
* on(cluster, volume,svm) group_left(node) volume_labels
```

**Reply by s-kuchi** [reply]
Perfect. Thank you @rahulguptajss
